plugin/filter.date.js

```diff
@@ -0,0 +1,3 @@
+import Vue from 'vue'
```

Comment:
what the purpose of that filter? where this is used?

Reply:
Folder plugin out to date. Removed